ntbs-service/Pages/Notifications/Edit/Episode.cshtml

```diff
@@ -51,77 +51,55 @@
-                    </label>
-                    <span nhs-span-type="ErrorMessage" id="tb-service-error" has-error="@TBServiceCodeError"
-                        ref="errorFieldTBServiceCode" asp-validation-for="Episode.TBServiceCode"></span>
+    
+    <cascading-dropdowns model="Episode" shouldvalidatefull="@fullValidation" inline-template>
+        <div>
+            @{
+                var TBServiceCodeError = !Model.IsValid("Episode.TBServiceCode");
+                var TBServiceCodeGroupState = TBServiceCodeError ? Error : Standard;
+                var TBServiceCodeSelectErrorState = TBServiceCodeError ? SelectType.Error : SelectType.Standard;
+            }
+            <nhs-form-group ref="formGroupTBServiceCode" nhs-form-group-type=@TBServiceCodeGroupState>
+                <label nhs-label-type="Standard" asp-for="Episode.TBServiceCode">
+                    Notifying TB service (required)
+                </label>
+                <span nhs-span-type="ErrorMessage" id="tb-service-error" has-error="@TBServiceCodeError"
+                    ref="errorFieldTBServiceCode" asp-validation-for="Episode.TBServiceCode"></span>
+                @if(Model.Notification.NotificationStatus == NotificationStatus.Draft) 
+                {
                     <select ref="selectFieldTBServiceCode" nhs-select-type="@TBServiceCodeSelectErrorState" asp-for="Episode.TBServiceCode"
                             asp-items="Model.TBServices" v-on:change="function (ev) {fetchHospitalList(ev); validate('TBServiceCode')(ev);}">
                     </select>
-                </nhs-form-group>
-                <br />
-                @{
-                    var HospitalError = !Model.IsValid("Episode.HospitalId");
-                    var HospitalGroupState = HospitalError ? Error : Standard;
-                    var HospitalSelectErrorState = HospitalError ? SelectType.Error : SelectType.Standard;
                 }
-                <nhs-form-group ref="formGroupHospitalId" nhs-form-group-type=@HospitalGroupState>
-                    <label nhs-label-type="Standard" asp-for="Episode.HospitalId">
-                        Hospital (required)
-                    </label>
-                    <span nhs-span-type="ErrorMessage" id="hospital-error" has-error="@HospitalError"
-                        ref="errorFieldHospitalId" asp-validation-for="Episode.HospitalId"></span>
-                    <select ref="selectFieldHospitalId" nhs-select-type="@HospitalSelectErrorState" asp-for="Episode.HospitalId"
-                            asp-items="Model.Hospitals" v-on:change="function(ev) { validate('HospitalId')(ev); }">
-                        <option value="" selected>Select Hospital</option>
+                else
+                {
+                    <select asp-for="Episode.TBServiceCode" nhs-select-type="Standard" disabled>
+                        <option selected="selected">@Model.Notification.Episode?.TBService?.Name</option>

```

Comment:
Ticket talks about text here doesn't it - might be worth verifying everyone will be happy with a disabled field, otherwise we could have this as hidden (for the sake of js not crying) and display text